config/external.yaml

```diff
@@ -1,3 +1,6 @@
 protocol-rack:
   url: https://github.com/socketry/protocol-rack
   command: bundle exec bake test
+sinatra:
+  url: https://github.com/sinatra/sinatra
+  command: bundle exec rake test
```

Comment:
There is also https://github.com/sinatra/sinatra/tree/main/sinatra-contrib and https://github.com/sinatra/sinatra/tree/main/rack-protection – they have their own tests. I don't know if they should be tested here too, just letting you know.

Reply:
It's fine to add more external tests as long as they can run without a huge amount of extra scaffolding.

Reply:
AFAIK `bake-test-external` doesn't support `Gemfile` in subfolder yet.

Reply:
ok, I'll check

Reply:
There was a bug, and I've fixed it and released in v0.5.0

Reply:
I'll open separate PR to onboard those as well.